content/klc/G2.1.adoc

```diff
@@ -1,21 +1,47 @@
 +++
-brief = "Defining generic and atomic parts"
+brief = "Generic, fully specified and atomic parts."
 +++
 
-In general terms, the _symbols_ in the KiCad library can be categorized into two groups:
+KiCad is flexible with how symbols and footprints can be connected. In general there are at least two workflows:
+. Generic symbol: Assign the footprint after (or during) symbol placement in eeschema.
+.. Footprint filters may be used to reduce the set of possible footprints.
```

Comment:
give an example like resistor symbol "R", filter "R_*"

Reply:
As i would keep the more detailed sections below and these have examples already in them, i think it is better to keep the introduction short.